From: Debian MySQL Maintainers <pkg-mysql-maint@lists.alioth.debian.org>
Date: Thu, 10 Aug 2017 20:40:28 +0200
Subject: innodb_simulate_comp_test_speedup
Description: Add options to innodb test that fixes possible test failure
 It was noticed that the test innodb_simulate_comp_failures_small
 repeatedly fails on some machines because the test is way too I/O
 heavy and mysqld starts to time-out under heavy load.
 .
 Remove this once upstream has released an official fix that includes
 --innodb-flush-log-at-trx-commit=2
Author: Otto Kekäläinen <otto@seravo.fi>
Bug: https://jira.mariadb.org/browse/MDEV-6682

---
 mysql-test/suite/innodb/t/innodb_simulate_comp_failures_small-master.opt | 1 +
 1 file changed, 1 insertion(+)

--- a/mysql-test/suite/innodb/t/innodb_simulate_comp_failures_small-master.opt
+++ b/mysql-test/suite/innodb/t/innodb_simulate_comp_failures_small-master.opt
@@ -1,2 +1,3 @@
 --innodb-file-per-table
+--innodb-flush-log-at-trx-commit=2
 
